hazelcast/src/main/java/com/hazelcast/cp/internal/MetadataRaftGroupManager.java

```diff
@@ -115,12 +119,12 @@
 
     // all fields below are state of the Metadata CP group and put into Metadata snapshot and reset while restarting...
     // these fields are accessed outside of Raft while restarting or local querying, etc.
-    @Probe
+    @Probe(name = CPSUBSYSTEM_METRIC_METADATA_RAFT_GROUP_MANAGER_GROUPS, unit = COUNT)
```

Comment:
Don't we use `COUNT` unit by default? Same question for the next probe here.

Reply:
Removed the unit.